plugin.video.mediathek/mediathek/kika.py

```diff
@@ -99,42 +104,73 @@ def buildVideoLink(self,pageLink):
       if("MP4 Web XL" in profile):
         links[3] = SimpleLink(directLink, 0);
 
+    date = None
+    date = time.strptime(unicode(self.regex_xml_time.search(xmlPage).group(1),"UTF-8"),u"%d.%m.%Y %H:%M");
     if(channel is not None):
-      return DisplayObject(channel,title,image,"",links,True, None);
+      return DisplayObject(channel,title,image,"",links,True, date);
     else:
-      return DisplayObject(title,"",image,"",links,True, None);
+      return DisplayObject(title,"",image,"",links,True, date);
 
   def buildPageMenu(self, link, initCount):
+    videoLinks = set()
     pageContent = self.loadPage(link);
     htmlPage =  BeautifulSoup(pageContent, 'html.parser')
+
     htmlElements = htmlPage.select(self.selector_videoPages)
-    videoLinks = set()
+    self.gui.log("found %d htmlElements"%len(htmlElements));
+    self.extractConfigLinks(videoLinks,pageContent);
+    self.extractVideoLinks(videoLinks,htmlElements);
 
+    if(len(videoLinks) == 0):
+      htmlElements = htmlPage.select(self.selector_allVideoPage)
+      self.extractVideoLinks(videoLinks,htmlElements);
+    count = initCount + len(videoLinks)
+
+    self.extractSubFolders(htmlPage,count);
+    displayObects = set()
```

Comment:
Typo :)

Reply:
thanks, i'am blind for that kind of mistakes...

Reply:
Not a problem, happens to us all